clevertap-core/src/main/java/com/clevertap/android/sdk/CTXtensions.kt

```diff
@@ -299,3 +305,17 @@ fun String?.toJsonOrNull(): JSONObject? {
         }
     }
 }
+
+fun View.applySystemBarsInsetsWithMargin(block : (bars:Insets, mlp:MarginLayoutParams) -> Unit) {
```

Comment:
why is there a need to create this extension function, seems to make the code unreadable where used.

Reply:
Resolving as discussed.